walletlibrary/src/main/java/com/microsoft/walletlibrary/did/sdk/VerifiableCredentialSdk.kt

```diff
@@ -69,7 +71,8 @@ internal object VerifiableCredentialSdk {
         polymorphicJsonSerializers: SerializersModule = Json.serializersModule,
         registrationUrl: String = "",
         resolverUrl: String = "https://discover.did.msidentity.com/v1.0/identifiers",
-        walletLibraryVersionInfo: String = ""
+        walletLibraryVersionInfo: String = "",
+        interceptors: List<HttpInterceptor> = emptyList()
```

Comment:
I think this is fine for now, but will be ever want to allow the developers to only add an httpInterceptor to certain network call. Does this allow for that? Also, are there any security implications with these interceptors?

Reply:
Sort of, thats why parameters are passed to identify the uri, method, and headers, but it does put the bulk of the request identifying work on developers. 
I also realized it should probably be a base class with no-op implementations so you only implement/override what you need.